app.py

```diff
@@ -15,7 +15,7 @@
 from resources.tenants import Tenants
 from resources.emergency_contacts import EmergencyContacts
 from flask_mail import Mail
-from resources.email import Email
+from resources.email import Email, InviteEmail
```

Comment:
Technically, email is not a resource. We can put the function on the email class.
```suggestion
from resources.email import Email
```